packages/main/src/SelectPopover.hbs

```diff
@@ -40,38 +42,57 @@
 		{{/unless}}
 
 		<ui5-list
-			selection-mode="SingleAuto"
+			mode="SingleSelectAuto"
 			separators="None"
 			@mousedown="{{_itemMousedown}}"
 			@ui5-item-click="{{_handleItemPress}}"
 		>
-			{{#each _syncedOptions}}
-				<ui5-li
-					id="{{this.id}}-li"
-					icon="{{this.icon}}"
-					?selected="{{this.selected}}"
-					?focused="{{this._focused}}"
-					tooltip="{{this.tooltip}}"
-					additional-text="{{this.additionalText}}"
-					?aria-selected="{{this.selected}}"
-					data-ui5-stable="{{this.stableDomRef}}"
-				>
-					{{this.textContent}}
-				</ui5-li>
+			{{#each options}}
+				{{#if this.isCustom}}
+					<ui5-li-custom
+						id="{{this._id}}-li"
+						tooltip="{{this.tooltip}}"
+						data-ui5-id="{{this._id}}"
+						icon="{{this.icon}}"
+						?selected="{{this.selected}}"
+						?focused="{{this.focused}}"
+						title="{{this.title}}"
+						additional-text="{{this.additionalText}}"
+						?aria-selected="{{this.selected}}"
+						data-ui5-stable="{{this.stableDomRef}}">
+						<slot name="{{this._individualSlot}}"></slot>
+					</ui5-li-custom>
+				{{else}}
+					<ui5-li
+						id="{{this._id}}-li"
+						tooltip="{{this.tooltip}}"
+						data-ui5-id="{{this._id}}"
+						icon="{{this.icon}}"
+						?selected="{{this.selected}}"
+						?focused="{{this.focused}}"
+						title="{{this.title}}"
+						additional-text="{{this.additionalText}}"
+						?aria-selected="{{this.selected}}"
+						data-ui5-stable="{{this.stableDomRef}}">
+						<slot name="{{this._individualSlot}}"></slot>
+					</ui5-li>
+				{{/if}}
 			{{/each}}
 		</ui5-list>
 	</ui5-responsive-popover>
 {{/if}}
 
 {{#if shouldOpenValueStateMessagePopover}}
 	<ui5-popover
+		{{!-- tabindex -1 is needed to keep the popover out of tab chain --}}
```

Comment:
Do you remember how the component should act if the value state contains link inside it? should it be focusable?

Reply:
The concept says that the first item (the header that containing the value help) in the dropdown is to be focusable and when you press tab while on it, the focus goes inside the value help to the first interactive item. This was never implemented in the select as the focus never leaves the root element (the select's div[role=combobox]). It needs to be clarified with keyboard spec first for a custom shortuct to move the focus to the value help, perhaps.

Reply:
removed the tabindex, will be fixed by https://github.com/SAP/ui5-webcomponents/pull/8955/